macros/PetBox_nest.config.mac

```diff
@@ -0,0 +1,42 @@
+### GEOMETRY
+/Geometry/PetBox/visibility true
+/Geometry/PetBox/tile_type_d HamamatsuVUV
+/Geometry/PetBox/tile_type_c HamamatsuVUV
+/Geometry/PetBox/single_tile_coinc_plane 0
+/Geometry/PetBox/add_teflon_block 1
+
+/Geometry/PetBox/tile_vis true
+/Geometry/PetBox/tile_refl 0.
+/Geometry/PetBox/sipm_time_binning 5. picosecond
+/Geometry/PetBox/sipm_pde 0.3
+
+### GENERATOR
+/Generator/IonGenerator/region SOURCE
+/Generator/IonGenerator/atomic_number 11
+/Generator/IonGenerator/mass_number 22
+#/Generator/SingleParticle/particle geantino
+#/Generator/SingleParticle/min_energy 1  eV
+#/Generator/SingleParticle/max_energy 1 keV
+#/Generator/SingleParticle/region CENTER
+
+/Actions/PetSensorsEventAction/min_charge 50
+
+### PHYSICS
+#/process/optical/processActivation Scintillation true
```

Comment:
When we use nest, this line should be set to `false`.

Reply:
Even though it is commented?

Reply:
It should not be commented!

Reply:
You're right! In the cluster when I perform the simulations it is not commented. I must have accidentally left it like that after a test in my local. Now it is fixed.